.travis.yml

```diff
@@ -8,7 +8,7 @@ rvm:
 
 before_install: gem install bundler
 
-script: bin/travis
+script: ci/travis
```

Comment:
I would suggest to get rid of `bin/travis` and use the script block as following:

```yml
script:
  - bundle exec rspec
  - bundle exec rubocop
```

Reply:
This would also remove the need for the `env.matrix:`-block

Reply:
Given the trivial `travis` script, I fully agree. (Hadn't looked at it in detail … 😄)

Reply:
Hmm … the only issue is that after that change we are left with a single Travis build and not two that clearly separate rspec from rubocop.

Reply:
IMHO, for a repo in size of the `request_headers_logger` it's not that critical

Reply:
We could do something like this: https://github.com/fidor/request_headers_logger/compare/master...buehmann:fix/bin-travis-alternative – But looking at it now, I am not fully convinced this is an improvement.

Reply:
Keep it simple.
Most Projects out there don't have a complex setup in Travis Yaml - why should we?

```yaml
script:
- bundle exec rspec
- bundle exec rubocop
```

I don't think we need to have separate rubocop jobs (especially when we test against more then one ruby version).

From the Travis Documentation:
> When one of the build commands returns a non-zero exit code, the Travis CI build runs the subsequent commands as well, and accumulates the build result.

Reply:
Kept it simple. 😉